backend/routers/v1/inventory.py

```diff
@@ -46,4 +48,33 @@ async def update_snack_route(sku: str, updates: SnackUpdateSchema):
     
 @router.delete("/snacks/{sku}", response_model=Snack)
 async def delete_snack_route(sku: str):
-    return delete_snack(sku)
\ No newline at end of file
+    return delete_snack(sku)
+
+
+@router.post("/snacks/bulk",response_model=BulkSnackResponse) 
+async def create_bulk_route(request:BulkSnackCreate):
+    
+    try:
+       #**# if len(bulk) == 1:
+           # raise Exception("Register a snack rather than a bulk")
+       # elif len(bulk)< 0:
+        #    raise Exception("Items inputted needs to be a positive number")
+        #Make a bulk_items variable that takes in the helper function then put that in the Bulk snack response to return it 
+        bulk_items=create_bulk_items(request.items)
+        print(bulk_items)
+
+        
+        if len(request.items) == 1:
+            raise Exception("Register a snack rather than a bulk")
+        elif len(request.items) < 0:
+            raise Exception("Items inputted need to be a positive number")
+       
+        return BulkSnackResponse(
+            success=True,
+            items=bulk_items,
+            error=None
+        )
+        
+    except Exception as e:
+
+        return BulkSnackResponse(success=False,items=None,error=f"Error Processing bulk request:{str(e)}")
```

Comment:
lets update the formatting to match the other return value
```suggestion
        return BulkSnackResponse(
               success=False,
               items=None,
               error=f"Error Processing bulk request:{str(e)}"
        )